src/rviz/default_plugin/markers/text_view_facing_marker.cpp

```diff
@@ -58,7 +58,10 @@ void TextViewFacingMarker::onNewMessage(const MarkerConstPtr& old_message, const
   ROS_ASSERT(new_message->type == visualization_msgs::Marker::TEXT_VIEW_FACING);
 
   if (new_message->text.find_first_not_of(" \t\n\v\f\r") == std::string::npos)
+  {
+    delete text_;
```

Comment:
Without having tested it:

```suggestion
    delete text_;
    text_ = nullptr;
```

Reply:
Thanks, good catch @v4hn 

I'm not sure about the c++ version we are targeting here, I think melodic is only systems that default to c++14 but as I didn't see anything specific in CMakeLists.txt I would probably do `text_ = 0;`

```suggestion
    delete text_;
    text_ = 0;
```

Reply:
I found the line where c++11 is enabled so `nullptr` is fine, I also added a line to reset the `MarkerSelectionHandler`